docs/geedocs/docsrc/BUILD_DOCS.md

```diff
@@ -145,10 +143,12 @@ branch and master.
       Keep the change, which has |release| instead of 5.3.3. This will help
       update the documentation based on the current Open GEE release.
 
-      Merges may become tricky if there are conflicts in the generated HTML.
+      NOTE: Merges may become tricky if there are conflicts in the generated HTML.
       If bringing 5.3.2 changes into master there will be 5.3.2
```

Comment:
consider rewording to make it version generic. May want to edit as "If bringing for example 5.3.2"... or some thing similar